trl/trainer/orpo_trainer.py

```diff
@@ -357,6 +356,22 @@ def make_inputs_require_grad(module, input, output):
                 "Your `Trainer` does not have an `accelerator` object. Consider upgrading `transformers`."
             )
 
+        # Import Liger loss if enabled
```

Comment:
IMO, passing `self.args.use_liger_loss` without liger installed should raised an error.